### DISCUSSION TITLE
Add the full configuration page to LXC Notes instead of just link to your page.

### DISCUSSION THREAD

**Discussion by Thedougler** [tteck/Proxmox#1845] [Ideas]
Would it be possible to modify your helper scripts to add the full description of each Container instead of just a link to your page? 

So when using the Homepage LXC helper script for example, upon successful completion it currently adds the following which is useful

# Homepage LXC
  ### https://tteck.github.io/Proxmox/
  <a href='https://ko-fi.com/D1D7EP4GF'><img src='https://img.shields.io/badge/☕-Buy me a coffee-red' /></a>

But It would be even more useful if it added the following expanded information instead:

---

# Homepage LXC
[Homepage](https://github.com/benphelps/homepage) is a self-hosted dashboard solution for centralizing and organizing data and information.

To create a new Proxmox VE Homepage LXC, run the command below in the Proxmox VE Shell.
To Update Homepage, run the command below in the LXC Console.

~~~
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/homepage.sh)"
~~~
⚡ Default Settings: 1GB RAM - 3GB Storage - 2vCPU ⚡
[Configuration](https://github.com/benphelps/homepage/wiki) (bookmarks.yaml, services.yaml, widgets.yaml) path: 
~~~
/opt/homepage/config/
~~~
### https://tteck.github.io/Proxmox/
  <a href='https://ko-fi.com/D1D7EP4GF'><img src='https://img.shields.io/badge/☕-Buy me a coffee-red' /></a>

**Comment by tteck** [comment]
The description function is automated using the `${APP}` variable and is employed in generating all LXC containers. Therefore, to address your inquiry, it's not possible to do so without making extensive code modifications. I opted for reusable scripts to minimize the need for ongoing maintenance.

The code snippet below is from the `build.func` script that generates the text in the "Notes".

```
# This function sets the description of the container.
description() {
  IP=$(pct exec "$CTID" ip a s dev eth0 | awk '/inet / {print $2}' | cut -d/ -f1)
  pct set "$CTID" -description "# ${APP} LXC
  ### https://tteck.github.io/Proxmox/
  <a href='https://ko-fi.com/D1D7EP4GF'><img src='https://img.shields.io/badge/☕-Buy me a coffee-red' /></a>"
  if [[ -f /etc/systemd/system/ping-instances.service ]]; then
    systemctl start ping-instances.service
  fi
}
```

**Reply by Thedougler** [reply]
Ahh I see, that's unfortunate but makes sense. Guess I'll just edit them after the fact.

**Reply by towerhand** [reply]
Could you add the OS selected during the installation to the notes? For example, Homepage LXC - Debian 12.

**Reply by tteck** [reply]
That information is shown in the console already.
```
root@homeassistant-core:~# exit
logout

Debian GNU/Linux 12 homeassistant-core tty1

homeassistant-core login: root (automatic login)

Home Assistant-Core LXC provided by https://tteck.github.io/Proxmox/

Last login: Tue Sep 26 20:31:58 EDT 2023 on tty1
root@homeassistant-core:~# 
```
